src/main/resources/assets/paths/progressReport.yaml

```diff
@@ -0,0 +1,35 @@
+post:
+  summary: Create a new DAR progress report
+  description: Create a new DAR progress report
+  tags:
+    - Data Access Request
+  parameters:
+    - name: parentReferenceId
+      in: path
+      description: The referenceId of the Data Access Request
+      required: true
+      schema:
+        type: string
+  requestBody:
+    required: true
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            dar:
+              schema:
+                $ref: '../schemas/DataAccessRequest.yaml'
```

Comment:
The more I read, the more I think this isn't allowed in the spec. I am fine with adding documentation to describe this case.